### PR TITLE
Fix typos: update registerCidrsetMetrics to registerCIDRSetMetrics

### PR DESCRIPTION
**Pull request by ii2day**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
update func name registerCidrsetMetrics to registerCidrSetMetrics
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Comment by ty-dc** [comment]
thanks.
/ok-to-test

File: pkg/controller/nodeipam/ipam/multicidrset/multi_cidr_set.go [hunk 1]
```diff
@@ -133,7 +133,7 @@ func NewMultiCIDRSet(cidrConfig *net.IPNet, perNodeHostBits int) (*MultiCIDRSet,
 	}
 
 	// Register MultiCIDRSet metrics.
-	registerCidrsetMetrics()
+	registerCidrSetMetrics()
```
Comment (danwinship) [inline pkg/controller/nodeipam/ipam/multicidrset/multi_cidr_set.go:136]: If the type is `MultiCIDRSet` then this function ought to be `registerCIDRSetMetrics`.
(In general, golang style is to never _partially_ uppercase acronyms within names; they are always either all-lowercase or all-uppercase.)

Comment (ii2day) [reply pkg/controller/nodeipam/ipam/multicidrset/multi_cidr_set.go]: OK, thank you, it has been modified to CIDR

**Comment by danwinship** [comment]
/approve
/lgtm
thanks! note that this won't merge for a few days until the 1.29 freeze ends

**Review by justankiit — COMMENTED** [review]
Thanks for Contributing here! lgtm!

**Comment by aojea** [comment]
hmm, I think this multiCIDRSet controller has to removed after https://github.com/kubernetes/kubernetes/commit/c2d473f0d438cedab2f1831d23457d24961e0f4e , I may oversight it

**Comment by ii2day** [comment]
> hmm, I think this multiCIDRSet controller has to removed after [c2d473f](https://github.com/kubernetes/kubernetes/commit/c2d473f0d438cedab2f1831d23457d24961e0f4e) , I may oversight it

It's great to hear from you all, this is my first contribution to the k8s community and I'm not quite sure what to do next? Should this pr be merged in or closed? thx

**Comment by aojea** [comment]
you have to wait until 1.29 is released, but this is good to go, thanks for contributing to the project

**Comment by ii2day** [comment]
> you have to wait until 1.29 is released, but this is good to go, thanks for contributing to the project

Okay, thanks again

**Pull request merged**